core/commands/swarm.go

```diff
@@ -1057,3 +1088,20 @@ func filtersRemove(r repo.Repo, cfg *config.Config, toRemoveFilters []string) ([
 
 	return removed, nil
 }
+
+func updateAndPersistConfig(env cmds.Environment, update func(*config.Config)) error {
+	r, err := fsrepo.Open(env.(*commands.Context).ConfigRoot)
+	if err != nil {
+		return fmt.Errorf("error opening repo: %w", err)
+	}
+	defer r.Close()
+	cfg, err := r.Config()
+	if err != nil {
+		return fmt.Errorf("error fetching config file: %w", err)
+	}
+	update(cfg)
+	if err := r.SetConfig(cfg); err != nil {
+		return fmt.Errorf("error removing peers from repo config: %w", err)
```

Comment:
If this method is generic for any config param, we should update this error message.